### PR TITLE
Layout cleanup

### PR DESCRIPTION
**Pull request by davidrapson**
Extracts out just the stylesheet cleanup and adjustments to functional page breakpoints from https://github.com/guardian/membership-frontend/pull/18

- Corrects breakpoints for functional pages based on Ben W designs:
- Removes dead code from old homepage
- Move nav/ into components/nav/
- Move _tones.scss to theme/
- Move _tier to components (slowly move styles out of pages)
- Add table of contents to style.scss and clean up imports

**Functional layout before**
screen shot 2015-01-05 at 10 59 58

**Functional layout after**
screen shot 2015-01-05 at 11 00 08

### CODE REVIEW

**Comment by davidrapson** [comment]
Taking this down as the build keeps failing on generating the icons sprite.

**Comment by davidrapson** [comment]
Re-opening. Found source of build failure, `icons/` now only contains sprite file so build fails as directory doesn't exist yet. Moved `icons.scss` to `stylesheets/` root.

**Comment by wpf500** [comment]
:+1:

File: frontend/assets/stylesheets/components/_page.scss [hunk 1]
```diff
@@ -2,6 +2,8 @@
    Page
    ========================================================================== */
 
+$_page-offset: rem(gs-span(2) + $gs-gutter);
```
Comment (feedmypixel) [inline frontend/assets/stylesheets/components/_page.scss]: I see the convention here. Bit picky feels like $page-offset would be nicer?

File: frontend/assets/stylesheets/pages/DEPRECATED.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Do not add any new styles to pages/. All new styles should be built as stand-alone components, not tied to page specific usage.
```
Comment (feedmypixel) [inline frontend/assets/stylesheets/pages/DEPRECATED.txt]: Maybe worth dropping a mail to devs to say we are moving away from this also?

**Pull request merged**